spec/controllers/users_controller_spec.rb

```diff
@@ -1,14 +1,46 @@
 require 'rails_helper'
 
 describe UsersController, :type => :controller do
-  describe ".permit" do
-    it "returns the cleaned params" do
-      user_params = {id: 5, email: "a@a.com", password: "blahblah", password_confirmation: "blahblah", 
-       remember_me: true, pending_organisation_id: 5, superadmin: true}
+  let(:user) { FactoryGirl.create(:user) }
```

Comment:
To have a user id which can be sent as a parameter later during testing edit action. Should I remove it to the relevant test case?